sdk/client/resourcemanager/poller_helpers_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package resourcemanager
```

Comment:
Would it be better to use a test package for these? I might have missed something but it doesn't seem like we're testing any unexported functions?

Reply:
Possibly, I don't think we particularly gain anything from doing that here though, so I'd question the need for that?